docs/visualization/index.md

```diff
@@ -1 +1,27 @@
 # Visualization
+
+## Basic APIs Concept
+
+- [Basic APIs Concept](./basic_api_concept.md) provided by CARET
+
+## Provided APIs
+
+### APIs in Plot Class
+
+- create_callback_frequency_plot
+- create_callback_jitter_plot
```

Comment:
create_callback_jitter_plot is currently deprecated.
It's ok to delete this line.
```suggestion
```

Reply:
I deleted the line.